react-ui/web/src/components/communication/contacts/Container.js

```diff
@@ -540,6 +545,10 @@ function Container (props) {
         setOpenSimPrefixes(!is_sim_prefixes_open);
     };
 
+    const modal_handler = (event) =>{   
+        !modal_state ? set_modal_state(true) : set_modal_state(false);
```

Comment:
boolean na yung modal_state di ba? I-simplify na lang na
```suggestion
        set_modal_state(!modal_state)
```
Paki-check kung tama yung suggestion